### ISSUE TITLE
cursor_command (aka --symbol-info/-U) seems broken.

### ISSUE THREAD

**Issue by tillt** [rampage644/sublime-rtags#29]
When trying to use the cursor_command, the resulting panel is showing one letter lines.

rc output:
```
/Users/till/Development/mesos-private/src/authentication/cram_md5/authenticatee.cpp:88:5:	    discarded(); // Fail the promise.
SymbolName: void CRAMMD5AuthenticateeProcess::discarded()
Kind: MemberRefExpr
Type: <bound member function type>
SymbolLength: 9
Range: 88:5-88:14
sizeof: 1
alignment (bytes): 4
Targets:
    /Users/till/Development/mesos-private/src/authentication/cram_md5/authenticatee.cpp:317:8:	  void discarded()
```

displayed panel in sublime 3:
```
/
U
s
e
[...]
```

**Comment by tillt** [comment]
Got a PR in the works for fixing this - see https://github.com/tillt/sublime-rtags for more.

**Comment by tillt** [comment]
Finally got around creating that PR - sorry for the delay - crazy busy as usual.
@rampage644  let me know what you think :)

**Comment by rampage644** [comment]
I've just merged #30 and close this.
Feel free to reopen on bugs.
